tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Output/Printing/SwiftPrinter.swift

```diff
@@ -148,3 +155,28 @@ internal class SwiftPrinter: Printer {
         }
     }
 }
+
+// MARK: - Reflection Helpers
```

Comment:
Nitpick: that looks like fundamental protocol/extensions that could be gathered in a separate file outside of `SwiftPrinter`.

Reply:
Done in https://github.com/DataDog/dd-sdk-ios/pull/386